### PR TITLE
Release/1.1.0

### LINKED ISSUE #73: Release 1.1.0
### Describe the bug

This issue is for tracking changes for the 1.1.0 release. Target release date: **March 2022 or Early April**.

## Pre-release steps
- [x] Confirm build commands, ensure GitHub Actions building plugin correctly

### [Release steps](https://github.com/10up/publisher-media-kit/blob/develop/CONTRIBUTING.md#release-instructions)
- [x] Branch: Starting from `develop`, cut a release branch named `release/1.1.0` for your changes.
- [x] Version bump: Bump the version number in `package-lock.json`, `package.json`, `publisher-media-kit.php`, `readme.txt`, and any other relevant files if it does not already reflect the version being released.  Update both the plugin "Version:" property and the plugin `PUBLISHER_MEDIA_KIT_VERSION` constant in `publisher-media-kit.php`.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` file with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Readme updates: Make any other readme changes as necessary. `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] Make sure the release date added in the `CHANGELOG.md`.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk` (`git checkout trunk && git merge --no-ff develop`).  `trunk` contains the stable development version.
- [x] Push: Push your `trunk` branch to GitHub (e.g. `git push origin trunk`).
- [x] Release: Create a [new release](https://github.com/10up/publisher-media-kit/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch.  Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the closed issues on the [milestone](https://github.com/10up/publisher-media-kit/milestone/3?closed=1).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/publisher-media-kit/actions/workflows/dotorg-push-deploy.yml) to finish deploying to the WordPress.org repository.  If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/publisher-media-kit/.  This may take a few minutes.
- [x] Close milestone: Edit the [milestone](https://github.com/10up/publisher-media-kit/milestone/3) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.1.0` do not make it into the release, update their milestone to`1.2.0` or `Future Release`

### PR DESCRIPTION
**Pull request by faisal-alvi**
### Description of the Change

Release and version bump to `1.1.0`! 🎉

Closes #73

### Alternate Designs

n/a

### Possible Drawbacks

none identified.

### Verification Process

Verified manually via any IDE and the GitHub Desktop UI.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Credits

Props @

### CODE REVIEW

**Comment by jeffpaul** [comment]
Note that I'm hoping to get some help on #72 this week, so holding off to see if that can get resolved and into this release

**Comment by faisal-alvi** [comment]
@jeffpaul ready for your review!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,30 @@
 
 All notable changes to this project will be documented in this file, per [the Keep a Changelog standard](http://keepachangelog.com/).  Moving forward, this project will (more strictly) adhere to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [1.1.0] - 2022-03-24
+### Added
+- Added mobile styling and fixed linting errors (props [@cldhmmr](https://github.com/cldhmmr), [@faisal-alvi](https://github.com/faisal-alvi), [@sudip-10up](https://github.com/sudip-10up), [@jeffpaul](https://github.com/jeffpaul) via #72)
```
Comment (jeffpaul) [inline CHANGELOG.md]: Please include https://github.com/treykane as well

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,30 @@
...
+
+### Changed
+
```
Comment (jeffpaul) [inline CHANGELOG.md]: Probably worth adding https://github.com/10up/publisher-media-kit/pull/81 once that gets merged as well

**Review by jeffpaul — APPROVED** [review]

**Pull request merged**